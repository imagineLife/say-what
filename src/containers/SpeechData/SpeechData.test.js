import React from 'react';
import ReactDOM from 'react-dom';
import { SpeechData } from './index';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import localStorage from 'mock-local-storage';

Enzyme.configure({ adapter: new Adapter() });

global.window = {};
window.localStorage = global.localStorage;

describe('SpeechData.js', () => {
	it('renders without crashing', () => {
	  shallow(<SpeechData />);
	});

	it('Renders <Redirect/> when not logged in', () => {
		const test = shallow(<SpeechData />);
		const instance = test.instance();
		const mockRedirect = test.find('Redirect');
		expect(mockRedirect.length).toEqual(1);
	})
})