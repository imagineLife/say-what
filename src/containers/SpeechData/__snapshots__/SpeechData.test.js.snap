// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SpeechData /> matches snampshot 1`] = `
<Redirect
  to="/login"
/>
`;
